### PR TITLE
Checkout submodules on test build too

### PR DESCRIPTION
**Pull request by jlvandenhout**
# Description of change

Currently the test build fails because we are in the process of switching to submodules for external repos, which when not checked out result in a lock file mutation when installing dependencies.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Bug fix (a non-breaking change which fixes an issue)

## How the change has been tested

The scripts are tested locally. The workflows should run in this PR and when merged.

## Change checklist

Tick the boxes that are relevant to your changes, and delete any items that are not.

- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have tested the wiki locally and tested that all external links work

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+name: test
+
+on:
+  workflow_dispatch:
```
Comment (Dr-Electron) [inline .github/workflows/test.yml:4]: Do we need a workflow_dispatch here? I mean it doesn't hurt to have it. Just asking if we want a clean setup or just keep it

File: package.json [hunk 2]
```diff
@@ -15,7 +15,9 @@
     "lint": "eslint --cache --fix src",
     "format": "prettier --write .",
     "prepare": "husky install",
-    "cli": "iota-wiki-cli"
+    "cli": "iota-wiki-cli",
+    "checkout": "git submodule update --init --recursive",
```
Comment (Dr-Electron) [inline package.json:19]: I though this doesn't work? Why do we keep it?

**Review by Dr-Electron — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (jlvandenhout) [reply package.json:19]: For local development it's still useful.

File: .github/workflows/test.yml (hunk 1 above)
Comment (jlvandenhout) [reply .github/workflows/test.yml:4]: Yeah you are right, we can remove this.

Comment (Dr-Electron) [reply .github/workflows/test.yml:4]:
```suggestion
```

Comment (Dr-Electron) [reply .github/workflows/test.yml:4]: I leave the honour of removing it to you :trollface:

File: package.json (hunk 2 above)
Comment (Dr-Electron) [reply package.json:19]: Hmm. Ok. I would include it in the cli tho. But we can leave it for now 👍

**Review by kilianhln — APPROVED** [review]

**Pull request merged**